zuul.d/jobs.yaml

```diff
@@ -0,0 +1,29 @@
+---
+- job:
+    name: install-yamls-centos-9-microshift-singlenode-podified-deployment
+    parent: base-microshift-openstack
+    nodeset: centos-9-microshift-single-node
+    timeout: 10800
+    required-projects:
+      - openstack-k8s-operators/install_yamls
+      - opendev.org/zuul/zuul-jobs
+      - openstack-k8s-operators/openstack-operator
+      - rdo-jobs
+      - rdo-infra/rdo-infra-playbooks
+    roles:
+      - zuul: rdo-jobs
+      - zuul: rdo-infra/rdo-infra-playbooks
+    run: ci/playbooks/deploy_podified_ctlplane.yaml
+    post-run: ci/playbooks/collect_logs.yaml
+    vars:
+      pv_storageclass: local-storage
+      overwrite_container_policy: true
+      setup_olm: true
+      standard_user: {}
+      pv_count:
+        start: 1
+        end: 12
+      pre_pull_images:
+        - registry.redhat.io/rhosp-rhel9/openstack-rabbitmq:17.0
```

Comment:
is this correct to use a downstream image?

Reply:
In Zuul job, we remove pull secret in pre playbook once microshift deployment finishes. 
For rabbitmq deployment, we need above image, otherwise rabbitmq deployment fails. 
In order to download the image, we needs creds from pull secret. So we made this general var via which one can download the image which depends on pull secret creds.

So it is correct to use a downstream image due to https://github.com/openstack-k8s-operators/openstack-operator/blob/be3d3c851b83ce01e7f0f4cf8a9bf13e764901f1/pkg/openstack/rabbitmq.go#L81
```
rabbitmq.Spec.Image = "registry.redhat.io/rhosp-rhel9/openstack-rabbitmq:17.0"
```

Reply:
oh I have not checked that. I think we'll switch to https://quay.io/repository/podified-zed-centos9/openstack-rabbitmq?tab=tags soon. but lets adjust then in a follow up

Reply:
ya i filed a few days ago https://github.com/openstack-k8s-operators/openstack-operator/issues/235

once we fix that we wont need this workaround.